cmd/argocd-util/main.go

```diff
@@ -220,6 +221,7 @@ func NewImportCommand() *cobra.Command {
 				os.Exit(1)
 			}
 			config, err := clientConfig.ClientConfig()
+			errors.CheckError(err)
```

Comment:
day-0 bug?

Reply:
yeah, I think check was missing from the beginning.